rules/gcp_audit_rules/gcp_iam_service_accounts_sign_blob.py

```diff
@@ -0,0 +1,17 @@
+from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_walk
```

Comment:
`event.deep_walk("field"...`

Reply:
could you elaborate?)

Reply:
I guess you meant this https://github.com/panther-labs/panther-analysis/pull/1148#discussion_r1525857290